yarn/datadog_checks/yarn/yarn.py

```diff
@@ -257,6 +258,16 @@ def _get_app_tags(self, app_json, app_tags):
                 self.log.error("Invalid value {} for application_tag".format(yarn_key))
         return tags
 
+    def collect_version(self, rm_address):
+        data = self._rest_request_to_json(rm_address, YARN_NODES_PATH, [])
+        node_info = data['nodes']['node']
+        for metrics in node_info:
+            try:
+                version = metrics['version']
+                self.set_metadata('version', version)
```

Comment:
Are we potentially calling this more than once?

Reply:
Added a check for that.